onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
I don't understand why we stick to a base image that has a sticked version of mesos and the we let the user choose the ``MESOS_RELEASE`` and ``STORM_RELEASE``